.travis.yml

```diff
@@ -1,38 +1,41 @@
-language: java
+language: android
 git:
   depth: 3
 matrix:
   include:
-  - os: linux
```

Comment:
I like this change, but could you keep linux build in the matrix too? I think it should be okay to publish from mac (I think you need that) but I'd like to keep the linux build going since I build boringssl here.

Reply:
Took a bit of wrangling, but I think this will work.
We're set to publish from linux only; but can easily change if needed.